src/Dev/Command/ConfigAudit.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace SilverStripe\Dev\Command;
+
+use SilverStripe\Core\ClassInfo;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\PolyExecution\PolyOutput;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+
+/**
+ * Command to audit the configuration.
+ * Can be run either via an HTTP request or the CLI.
+ */
+class ConfigAudit extends DevCommand
+{
+    protected static string $commandName = 'config:audit';
+
+    protected static string $description = 'Find configuration properties that are not defined (or inherited) by their respective classes';
+
+    private static string|array $permissions_for_browser_execution = [
```

Comment:
```suggestion
    private static array $permissions_for_browser_execution = [
```

Reply:
Good catch, done. There was an older iteration where I did allow string - but I decided later that's silly.